### PR TITLE
Add to_no_content

### PR DESCRIPTION
**Pull request by kujma10**

### CODE REVIEW

File: lib/salestation/web/responses.rb [hunk 1]
```diff
@@ -13,6 +13,10 @@ def self.to_ok
         -> (object) { Deterministic::Result::Success(Responses::OK.new(body: object)) }
       end
 
+      def self.to_no_content
+        ->  proc { Deterministic::Result::Success(Responses::NoContent.new(body: {})) }
```
Comment (indrekj) [inline lib/salestation/web/responses.rb]: `-> (*) {}` looks better

Comment (kujma10) [reply lib/salestation/web/responses.rb]: https://github.com/salemove/salestation/pull/20/commits/e3b584b0506c1ae3b2984c8b0686478b02bb0358

File: lib/salestation/web/responses.rb [hunk 2]
```diff
@@ -13,6 +13,10 @@ def self.to_ok
         -> (object) { Deterministic::Result::Success(Responses::OK.new(body: object)) }
       end
 
+      def self.to_no_content
+        ->  (*) { Deterministic::Result::Success(Responses::NoContent.new(body: {})) }
```
Comment (indrekj) [inline lib/salestation/web/responses.rb]: human lint: two spaces between -> and (

Comment (kujma10) [reply lib/salestation/web/responses.rb]: Good eye https://github.com/salemove/salestation/pull/20/commits/5724141ded08d229c32fb915e91a4fdb7fb0e86c

There are bunch of rubocop errors in this project. Need to be fixed in a separate PR.

**Comment by deiwin** [comment]
No test?

**Comment by kujma10** [comment]
Added a test

File: spec/salestation/web_spec.rb [hunk 3]
```diff
@@ -36,5 +36,15 @@ def status(*)
         expect(web_app.process(response)).to eql(JSON.dump(response_body.merge(debug_message: '')))
       end
     end
+
+    context 'when no content response' do
+      let(:response) {
+        Salestation::Web::Responses.to_no_content.call
```
Comment (indrekj) [inline spec/salestation/web_spec.rb]: changing this to `to_ok` should also pass... Also, maybe write a unit test instead. easier to test. this doesn't look correct place for it.

Comment (kujma10) [reply spec/salestation/web_spec.rb]: https://github.com/salemove/salestation/pull/20/commits/06307f32c449904c04f00c0dd0f0688e6eafe074

**Comment by SteamRiC** [comment]
While you are here, could you also add `return {} if body.empty?` before this line?
https://github.com/salemove/salestation/blob/master/lib/salestation/web/request_logger.rb#L75

When it returns no content, we currently get `Failed to parse response` logs.

**Pull request merged**